.github/workflows/msbuild.yml

```
@@ -112,6 +112,8 @@ jobs:
       uses: actions/checkout@v2
       with:
         path: dafny
+        submodules: true
+    - run: rm Test/libraries/lit.site.cfg
```

HA, cute. :)

```suggestion
    - run: rm dafny/Test/libraries/lit.site.cfg
```

(you just haven't changed into the `dafny` directory yet at this point)

Worth adding a comment here too as to why you have to do this.